.github/workflows/node.js.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        cache: 'npm'
+    - run: npm i --include=dev
```

Comment:
Can we use yarn here too? I'd prefer to make this consistent with the dev experience
https://github.com/Borales/actions-yarn

Reply:
I did my best O.O